### PR TITLE
Add a handler for Gogs webhooks.

### PR DESCRIPTION
**Pull request by crrobinson14**

### CODE REVIEW

**Comment by viktorbenei** [comment]
Looks perfect, thank you!

Anything else you want to add? If you have the time you should squash the commits, but other than that it seems mergeable ;)

**Comment by crrobinson14** [comment]
@viktorbenei This has been working really well for over a week. I think we're ready to merge!

**Comment by viktorbenei** [comment]
All right, I'll do that tomorrow, thanks a lot!!

**Comment by viktorbenei** [comment]
Released in [v1.1.4](https://github.com/bitrise-io/bitrise-webhooks/releases/tag/1.1.4), and it's now available on hooks.bitrise.io

Thanks again for the contribution!! ;)

**Comment by viktorbenei** [comment]
@crrobinson14 we just implemented Tag push triggers for every other service, but we don't have a Gogs server on hand - would you be open to implement it?

This is pretty much all what have to be done: https://github.com/bitrise-io/bitrise-webhooks/pull/24/commits/1befb4e29b65053a99cc9a27de6e6365997536f0#diff-7c846d329541006a89b4f8c8a6e2f0a0 

TL;DR; simply process `refs/tags/TAG` in addition to `refs/heads/BRANCH`, and that's all. The only thing what might have to be handled additionally is the case when the Tag was deleted - GitHub marks the webhook with a "deleted: true" property, other services usually just send the same event as when the tag was created but with a "0000...00" related commit hash.

Let us know if you'd need any help or if you don't have the time - and thanks again for your contribution!

**Comment by crrobinson14** [comment]
Happy to! Give me a few days - I'm traveling this weekend. I'm not sure how Gogs flags deleted tags but they must do something - I'll look into it.

**Comment by crrobinson14** [comment]
Since this comment thread is a merged PR, let's start a new issue to track the change? I'll file it, but I think you need to be the one to assign it to me.

**Comment by viktorbenei** [comment]
Sure, no rush at all ;)

We'll most likely still tweak a couple of things on the web UI anyway during the next days/weeks.
